extensions/applicationinsights-react-js/package.json

```diff
@@ -41,15 +41,15 @@
         "rollup-plugin-commonjs": "^9.3.4",
         "rollup-plugin-node-resolve": "^3.4.0",
         "rollup-plugin-replace": "^2.1.0",
-        "rollup-plugin-uglify": "^6.0.0",
         "rollup-plugin-cleanup": "3.2.1",
         "ts-jest": "^24.0.2",
-        "tslint": "^5.19.0",
-        "tslint-config-prettier": "^1.18.0",
-        "typescript": "^3.9.3"
+        "typescript": "^3.9.3",
+        "tslib": "^2.0.0",
+        "globby": "^11.0.0"
     },
     "dependencies": {
-        "@microsoft/applicationinsights-shims": "^1.0.3",
+        "@microsoft/dynamicproto-js": "^1.1.1",
+        "@microsoft/applicationinsights-shims": "^2.0.0",
```

Comment:
This won't compile until we publish v2.0.0 -- This PR currently includes the changes from #1523